blueman/plugins/applet/DiscvManager.py

```diff
@@ -31,21 +31,18 @@ class DiscvManager(AppletPlugin):
     }
 
     def on_load(self, applet):
-        self.item = create_menuitem(_("_Make Discoverable"), get_icon("edit-find", 16))
-        self.item_label = self.item.get_child().get_children()[1]
-        applet.Plugins.Menu.Register(self, self.item, 20, False)
+        self.item = applet.Plugins.Menu.add(self, 20, text=_("_Make Discoverable"), icon_name="edit-find",
+                                            tooltip=_("Make the default adapter temporarily visible"),
+                                            callback=self.on_set_discoverable, visible=False)
 
         self.Applet = applet
         self.adapter = None
         self.time_left = -1
 
-        self.item.connect("activate", self.on_set_discoverable)
-        self.item.props.tooltip_text = _("Make the default adapter temporarily visible")
-
         self.timeout = None
 
     def on_unload(self):
-        self.Applet.Plugins.Menu.Unregister(self)
+        self.Applet.Plugins.Menu.unregister(self)
```

Comment:
Adding to the menu was changed from ``Register`` to ``add``, do the same here and use ``remove``?